src/App/Hangouts/Update.elm

```diff
@@ -1,22 +1,93 @@
 module App.Hangouts.Update exposing (..)
 
 import Navigation
+import Paginate exposing (fromList, goTo, next, prev, first, last)
```

Comment:
I believe it would make more sense to just `import Paginate` and not expose anything, since looking at the rest of the code, we're qualifying every call to these functions with `Paginate`.